src/pill/types.ts

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export type Type = "outline" | "solid";
+export type ColorType = "black" | "grey" | "green" | "yellow" | "red" | "blue";
+
+export interface PillProps extends React.HTMLAttributes<HTMLDivElement> {
+    type: Type;
+    colorType: ColorType;
```

Comment:
If there is a default value, then it should not be mandatory, you also stated `mandatory: false` in the docs